### PR TITLE
Align round-corners-to-spec

### PR DESCRIPTION
**Pull request by patrick-tolosa**
#### What does this PR solve?
This PR aligns RoundCorners `r_` with our specifications.
In this PR we didn't use an interface `IRoundCorners`, and in general, they seem to be redundant, we might consider removing them in the future

#### Final checklist
- [x] JSdoc - Add proper docstrings based on our PHP implementation
- [x] JSdoc - Hide private functions using @private
- [x] Implementation is aligned with PHP Reference(If different, please specify why)
- [x] Tests - Add proper tests to the added code

### CODE REVIEW

File: src/actions/roundCorners/RoundCornersAction.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  }
+
+  /**
+   * @param {number} a
+   * @param {number} b
+   * @param {number} c
+   * @param {number} d
+   * @return {RoundCornersAction}
+   */
+  radius(a:number, b?:number, c?:number, d?:number):this {
```
Comment (maoznir) [inline src/actions/roundCorners/RoundCornersAction.ts:23]: Why not meaningful names?

Comment (patrick-tolosa) [reply src/actions/roundCorners/RoundCornersAction.ts:23]: It's four identical radiuses, so..

radiusA, radiusB, radiusC, radiusD?

Comment (maoznir) [reply src/actions/roundCorners/RoundCornersAction.ts:23]: As a user, how do I know how each one will affects the image? shouldn't their names and docstring convey that?

Comment (patrick-tolosa) [reply src/actions/roundCorners/RoundCornersAction.ts:23]: Added the PHP Docstrings and links to the reference

**Pull request merged**